simulator/simulator-view/src/shared/services/fire-creation.service.ts

```diff
@@ -28,13 +30,16 @@ export class FireCreationService {
     this.type = type;
     this.type.intensity = this.intensity;
     this.isSettingNewElement = true;
+    this.$isInCreationState.emit(this.isSettingNewElement);
```

Comment:
? non, à la limite tu fais this.$intensity.emit(this.intensity); mais pas ça. On ne devrait rentrer en creationState qu'au moment de lancer la fonction "create"